releasenotes/notes/qft-little-endian-d232c93e044f0063.yaml

```diff
@@ -0,0 +1,10 @@
+---
+upgrade:
+  - |
+    The QFT circuit in the circuit library now computes the Fourier transform
+    using a little-endian representation of tensors, i.e. the state
+    :math:`|1\rangle` maps to :math:`|0\rangle - |1\rangle + |2\rangle - ..`
+    assuming the computational basis correspond to little-endian bit ordering
+    of the integers. :math:`|0\rangle = |000\rangle, |1\rangle = |001\rangle`,
+    etc.
+
```

Comment:
Should this raise a runtime warning? I'm wondering about users who have built expectations around the current library big-endian QFT, then having their data silently change under them, which would be hard to track down.

Reply:
I would characterize the original behavior as a bug, and this as a bug fix. The original code did not compute the Fourier transform because counts, statevectors, everything in Qiskit is little endian. So whoever used it worked around the bug in some way. I think we should highlight the bugfix prominently in the release notes, but I don't think the code needs to keep referring to the old buggy behavior.